toolkit/resources/manifests/package/update_manifest.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
```

Comment:
We could probably do in the toolkit directory:
`git checkout 2.0-stable -- resources/manifests`

Reply:
This is my goto command as well, however there are issues with it unfortunately. If one of the packages needed for cycle resolution is updated (and the cycle has an explicit version requirement in int) the core repo will fail to build (since it expects the toolchain to provide the exact package version needed to break the cycle, and the version may not match).

I've actually been working on a set of documents to help one of our customers work in the core repo. Its stiill early draft but it might be of interest: 
- https://github.com/microsoft/CBL-Mariner/pull/5199
- https://github.com/microsoft/CBL-Mariner/pull/5218
- https://github.com/microsoft/CBL-MarinerTutorials/pull/37

Reply:
I have a section on getting stable commits here: https://github.com/microsoft/CBL-Mariner/blob/479bf4586a131a871bbdcd3f3a43e3e259e274c5/toolkit/docs/building/building.md#clone-and-sync-to-a-stable-commit

Reply:
Yes. force fetch actually solves the problem. Let me abandon this PR. But let the branch be here, so this code can be used later if required in some other scenario.